src/api/conversations.js

```diff
@@ -39,6 +38,27 @@ export class ConversationsApi extends BaseApi {
     }
 
     /**
+     * Send an image to an app user's conversation
+     * @param  {string} userId - an user id
+     * @param  {Blob|Readable stream} source - source image
```

Comment:
https://developer.mozilla.org/en/docs/Web/API/Blob
The File object inherits from it and it's used to send data to the server in a form.